core-web/libs/portlets/dot-analytics-search/portlet/jest.config.ts

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable */
 export default {
-    displayName: 'dot-analytics-search',
+    displayName: 'portlets-dot-analytics-search',
     preset: '../../../../jest.preset.js',
     setupFilesAfterEnv: ['<rootDir>/src/test-setup.ts'],
-    coverageDirectory: '../../../../coverage/libs/portlets/dot-analytics-search/portlet',
+    globals: {},
+    coverageDirectory: '../../../../../target/core-web-reports/',
```

Comment:
I believe the coverageDirectory is already defined in the base Jest configuration, so this attribute is not necessary.